libr/util/vector.c

```diff
@@ -24,10 +24,13 @@
 			vec->capacity = 0; \
 			break; \
 		} \
-		void **new_a = realloc (vec->a, vec->elem_size * new_capacity); \
+		void **new_a = calloc (1, vec->elem_size * new_capacity); \
```

Comment:
memleak here.

Please drop all changes to this file, and rebase your pr with our master branch. The problem of uninitialized memory of this macro was recently fixed

Reply:
awesome! rebased